app/views/puppetclasses/_class_parameters.html.erb

```diff
@@ -1,18 +1,62 @@
-<% value_hash  = Classification::ClassParam.new(:host=>obj).inherited_values if obj.class.model_name == "Host" %>
-<% lookup_keys.each_with_index do |key,i| %>
-    <tr id="puppetclass_<%= puppetclass.id %>_params[<%= key.id %>]">
-      <%= content_tag :td, (i == 0 ? {:rowspan => lookup_keys.size} : {:class => 'hide'}) do
-        # In order to use the class .hide-first-col, we must have an extra, invisible cell.
-        # Plus, we make the class name available from javascript without having to look at any previous row.
-        content_tag :span, puppetclass.name, :'data-property' => 'class'
-      end %>
-      <td>
-        <%= content_tag :span, key.key, :class => "#{"override-param" if key.overridden?(obj)}", :'data-property' => 'name' %></td>
-      <td><%= obj.class.model_name == "Host" ? host_key_with_diagnostic(obj, value_hash, key) : hostgroup_key_with_diagnostic(obj, key)%></td>
-      <td>
-          <%= link_to_function(_("override"), "override_class_param(this)", :title => _("Override this value"),
-                               :'data-tag' => 'override', :class =>"btn btn-default") if authorized_via_my_scope("host_editing", "create_params") && !key.overridden?(obj) %>
-      </td>
+<div id='<%= puppetclass.to_s.gsub(':', '_') + '_' + puppetclass.id.to_s %>' class='tab-pane fields'>
+  <% lookup_keys = overridable_lookup_keys(puppetclass, obj) %>
+  <% value_hash = Classification::ClassParam.new(:host => obj).inherited_values if obj.class.model_name == "Host" %>
+  <table class="table table-condensed">
+    <thead>
+    <tr>
+      <th class='col-md-2'><%= _('Name') %></th>
+      <th class='col-md-7'><%= _('Value') %></th>
+      <th><%= _('Actions') %></th>
     </tr>
-<% end %>
+    </thead>
+    <tbody>
+    <% lookup_keys.each_with_index do |key, i| %>
+        <tr class="lookup_key_row <%="overridden" if key.overridden?(obj) %>" data-lookup-key-id="<%=key.id%>">
+          <td class='col-md-2'>
+            <%= content_tag :span, key.key, :class => "col-md-2 lookup_key_name", :'data-property' => 'name' %></td>
+          <td class='col-md-7'><%= obj.class.model_name == "Host" ? host_key_with_diagnostic(obj, value_hash, key) : hostgroup_key_with_diagnostic(obj, key) %></td>
+          <td>
+            <%= link_to(_("override"), nil, :title => _("Override this value"),
+                                 :'data-tag' => 'override', :class => "btn btn-default override_button" ) if authorized_via_my_scope("host_editing", "create_params") %>
+            <%= content_tag :span, _("Overridden"), :class => "overridden_message" %>
+          </td>
+        </tr>
+        <tr id="lookup_value_for_<%=key.id%>" class="lookup_key_override_row <%="visible" if key.overridden?(obj) %>">
+          <% value = key.overriding_value(obj) %>
+          <td class='col-md-2'>
+            <%= content_tag :span, key.key, :class => "col-md-2 lookup_key_name", :'data-property' => 'name' %></td>
```

Comment:
leave this out perhaps?  It's already on the line above, we could instead have better styling to show the two rows (the original and the override) are one and the same.

e.g. the same background colour, or a little link like we have on the e-mail preferences of users:

![screenshot from 2015-05-22 13 01 26](https://cloud.githubusercontent.com/assets/482089/7770128/c43c06ee-0082-11e5-8740-fd09a2f1991e.png)